dkm/New-AdfsDkmContainer.ps1

```diff
@@ -0,0 +1,126 @@
+[CmdletBinding()] 

```

Comment:
Could you add basic Pester tests for at least the following scenarios. For more details, see the [tests on this project](https://github.com/Microsoft/adfsLogTools/blob/master/TESTDETAILS.md)

1. Create a DKM container and check it exists 
2. Create a DKM container and check it has the expected ACEs